ingestors/elm.go

```diff
@@ -0,0 +1,55 @@
+package ingestors
+
+import (
+	"fmt"
+	"github.com/librariesio/depper/data"
+	"github.com/mmcdole/gofeed"
+	"log"
+	"net/url"
+	"strings"
+	"time"
+)
+
+const ElmSchedule = "0 */4 * * *"
```

Comment:
these should be lowercased, no need to export.

Reply:
was copying rubygems...

Reply:
I did it bad!